dashboard/db/migrate/20170403190323_add_using_text_mode_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddUsingTextModeToUsers < ActiveRecord::Migration[5.0]
+  def change
+    add_column :users, :using_text_mode, :boolean
```

Comment:
yeah, I think null is OK. that just means you've never flipped the bit one way or the other.